chapters/planning-data-work.tex

```diff
@@ -517,21 +497,23 @@ \subsection{Documenting and organizing code}
 Even you will probably not remember the exact choices that were made in a couple of weeks.
 Therefore, you must document your precise processes in your code.
 
-Code organization means keeping each piece of code in an easily findable location.
+Code organization means keeping each piece of code in an easy-to-find location.
```

Comment:
```suggestion
Code organization means keeping each piece of code in an easy-to-find location and naming them in a meaningful way.
```
I think a reference back to file naming should be included in the definition of code organization